src/bin/post-comment-to-pr.py

```diff
@@ -34,11 +34,28 @@ def get_pull_request(pr_url):
     return pr_endpoint
 
 
-def post_comment(pr_endpoint, test_instance):
+def check_for_comment(pr_endpoint, body):
+    oauth_key = os.getenv('GITHUB_OAUTH_TOKEN')
+
+    headers = {
+        'Authorization': 'token {0}'.format(oauth_key),
+        'Accept': 'application/vnd.github.v3+json'
+    }
+
+    response = requests.get(pr_endpoint, headers=headers)
+
+    for comment in response.json():
+        if comment['body'] == body:
```

Comment:
Maybe we can match any comment created by this script on the PR, and update its contents every deployment? This would remove the outdated info if it was switched to another test instance. Then we can also add more info to the comment, like last deploy time, and perhaps other branches pinned to the instance at the moment of deploy.

Reply:
I made some adjustments if you want to have another look. It now finds the existing comment and updates its content. Tested in a personal repo. It looks like this and it tries to match the title (first line):

![rocket](https://user-images.githubusercontent.com/939357/116240760-deba7400-a76c-11eb-9a7e-a76d80446a93.png)